broker-daemon/orderbook/index.js

```diff
@@ -97,12 +101,12 @@ class Orderbook {
       this.synced = false
       watcher.removeListener('sync', onWatcherSync)
       watcher.removeListener('error', onWatcherError)
+      this._updateExponentialDelayPeriod()
```

Comment:
I'd prefer that a function like this return a value rather than impacting state as a side effect

Reply:
I don't think it's necessary to have the additional methods after refactoring the retry logic, so this is no longer impacting state as a side effect.

Question: Do you want to also implement exponential backoff within `onWatcherError`? If so, I think it makes sense to bring back that method and make it a pure function.

Reply:
I don't think we need backoff in `onWatcherError` just yet - the errors that trigger that are local (rather than remote) so it's unlikely that we'll need to back off of them.